jenkinsfiles/Jenkinsfile.deploy-builder

```diff
@@ -1,4 +1,7 @@
 library "contrailWindows@$BRANCH_NAME"
 
-prepareAndRunAnsiblePlaybook(config: prepareDeployBuilder,
-                             playbook: 'vmware-deploy-template.yml')
+prepareAndRunAnsiblePlaybook(playbook: 'vmware-deploy-template.yml',
+                             vm_role: 'builder',
```

Comment:
Do we really need to specify all these params to deploy? Arent they stored in template already?